src/viam/api/config/viam-cpp-sdk-libviamapi.pc.in

```diff
@@ -1,4 +1,4 @@
-prefix=@CMAKE_INSTALL_PREFIX@
+prefix=/usr
```

Comment:
This definitely can't be changed like this. Non-cmake projects need these files in order to use the C++ SDK, and it is still legitimate to install the SDK to somewhere that isn't `/usr`.

Reply:
reverted in 8f501a5